CONTRIBUTING.md

```diff
@@ -108,6 +108,85 @@ Example:
   void sequenceNumber(long sequenceNumber);
 ```
 
+## Adding new functionality without breaking APIs
+Ideally, we'd want to add new functionality without breaking existing APIs, especially within the scope of the API modules that are being checked by [Revapi](https://revapi.org/).
+
+Let's assume we'd want to add a `createBranch(String name)` method to the `ManageSnapshots` API.
+
+The most straight-forward way would be to add the below code:
+
+```java
+public interface ManageSnapshots extends PendingUpdate<Snapshot> {
+  // existing code...
+
+  // introduces an API-breaking change
```

Comment:
Minor nit: can we say

// introduce an API-breaking change by adding the method
or
// introduce an API-breaking change by adding the below method

Reply:
done